geo/Cargo.toml

```diff
@@ -13,10 +13,12 @@ rust-version = "1.65"
 categories = ["science::geo"]
 
 [features]
-default = ["earcutr"]
+default = ["earcutr", "delaunay", "voronoi"]
 use-proj = ["proj"]
 proj-network = ["use-proj", "proj/network"]
 use-serde = ["serde", "geo-types/serde"]
+delaunay = ["ndarray", "ndarray-linalg/openblas"]
```

Comment:
when we come to getting this working in our builds, I guess we're going to need some way to allow using the static linking option available at https://github.com/rust-ndarray/ndarray-linalg/blob/master/ndarray-linalg/Cargo.toml

Reply:
and https://github.com/blas-lapack-rs/openblas-src/issues/88 will make static linking much more tricky.

Reply:
ndarray and linalg dependencies have been removed